Makefile.am

```diff
@@ -122,7 +122,7 @@ recheck-memory:
 # must be from main, as only that has current and existing images; but testvm.py API is stable
 # support CI testing against a bots change
 bots:
-	tools/make-bots
+	test/common/make-bots
```

Comment:
I think this breaks all subprojects, so maybe retain a symlink?

Reply:
It does, but it's very easy to fix on upgrade.